File: PHP/car.php
<?php
require_once('account.php');
class Car
{
    public $id;
    public $license;
    public $driver;
    public $passenger;

    private function getClassName()
    {
        return get_called_class();
    }


    public function __construct($license, $driver)
    {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function PrintDataCar()
    {
        echo "Licencia: $this->license, Driver: {$this->driver->name},
        document: {$this->driver->document}";
    }
}

File: Python/car.py
# from driver import Driver
from account import Account


class Car(Account):
    id = int
    license = str
    driver = Account
    passengenger = int

    def __init__(self, license, driver):
        # super().__init__(name, document, email, password)
        self.license = license
        self.driver = driver
